lib/src/key/mod.rs

```diff
@@ -2,11 +2,11 @@
 ///
 /// KV              /
 ///
-/// ND              /!nd{nd}
-/// LQ              /!nd{nd}*{ns}*{db}!lq{lq}
-///
 /// HB              /!hb{ts}/{nd}
 ///
+/// ND              /!nd{nd}
```

Comment:
We have `/!nd` here but we are using `/!cl` which isn't in this list...

Reply:
Sur-253 will resolve this fully. I want to keep the diffs separate for this pr, if that's ok with you.

Reply:
https://github.com/surrealdb/surrealdb/pull/2257